ASP_tests.py

```diff
@@ -269,16 +267,22 @@ def test_extract_sn_from_parquet_file_and_write_to_csv():
     sn_ids = pd.read_csv(output_path, header=None).values.flatten()
     test_sn_ids = pd.read_csv(pathlib.Path(__file__).parent/"tests/testdata/test_snids.csv", header=None).values.flatten()
     assert np.array_equal(sn_ids, test_sn_ids), "The SNIDs do not match the test example"
-    
+
 
 def test_make_regular_grid():
     wcs = np.load('./tests/testdata/wcs_dict.npz', allow_pickle=True)
-    wcs = dict(wcs)
-    ra_center = wcs['CRVAL1']
-    dec_center = wcs['CRVAL2']
-    for key in wcs.keys():
-        wcs[key] = wcs[key].item()
-    wcs = galsim.wcs.readFromFitsHeader(wcs)[0]
+    # Loading the data in this way, the data is packaged in an array,
+    # this extracts just the value so that we can build the WCS.
+    wcs_dict = dict(wcs)
+    ra_center = wcs_dict['CRVAL1']
+    dec_center = wcs_dict['CRVAL2']
+    for key in wcs_dict.keys():
+        wcs_dict[key] = wcs_dict[key].item()
+    for i in range(2):
+        if i == 0:
+            wcs = snappl.wcs.GalsimWCS.from_header(wcs_dict)
+        else:
+            wcs = snappl.wcs.AstropyWCS.from_header(wcs_dict)
```

Comment:
This seems super-weird.  You're running a for loop where `i=0` and then `i=1`, and then you have an if statement for `i==0` and then an `else`?
a. Why not just write out the two lines?
b. But really, what is even the point?  The `else` block is always the last thing run in the loop.